lib/monitor.js

```diff
@@ -40,14 +41,28 @@ Monitor.prototype.createMonitor = function (opt) {
 };
 
 /**
+ * Utility function for converting object into array of tags in the
+ * ['key1:value1', 'key2:value2'] format
+ */
+ Monitor.prototype._toTags = function (json) {
+  return Object.keys(json).map(function(key) {
+    return key + ':' + json[key];
+  });
+};
+
```

Comment:
for kicks I dug around in datadog's docs for a second to see if I could find anything, but they don't actually say what they do with multiple colons. I'd chalk that up to user/developer error if a key or value has a colon in it and it doesn't work as they expect in datadog...